src/manipulation.js

```diff
@@ -2,8 +2,8 @@ define( [
 	"./core",
 	"./var/concat",
 	"./var/push",
+	"./var/rcheckableType",
 	"./core/access",
-	"./manipulation/var/rcheckableType",
```

Comment:
If you're going to rearrange here, you also need to do so in the parameter list.